README.md

```diff
@@ -117,6 +120,8 @@ MIT Licensed. See [LICENSE](https://github.com/terraform-aws-modules/terraform-a
 | worker_additional_security_group_ids | A list of additional security group ids to attach to worker instances | list | `<list>` | no |
 | worker_group_count | The number of maps contained within the worker_groups list. | string | `1` | no |
 | worker_groups | A list of maps defining worker group configurations. See workers_group_defaults for valid keys. | list | `<list>` | no |
+| worker_instance_profile_name | If provided, the EKS worker nodes will spawn using this EC2 instance IAM profile. If not given, an instance profile will be created with the necessary policies attached. | string | `` | no |
+| worker_instance_role_name | If provided, the EKS worker nodes will spawn using this EC2 instance IAM role. If not given, an instance role will be created with the necessary policies attached. | string | `` | no |
```

Comment:
Why pass the role name when really only the instance profile is used by the launch configuration?

Reply:
@max-rocket-internet The worker instance role name is used in [`aws_auth.tf` to pass into the kubeconfig](https://github.com/terraform-aws-modules/terraform-aws-eks/pull/121/files#diff-2812d2bbb29e6bacd1bed6f9acf4a797R27) and [worker-role.tpl](https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/templates/worker-role.tpl#L1)